From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aleksey Gavrilov <le9i0nx@gmail.com>
Date: Fri, 8 Dec 2017 06:43:54 +0000
Subject: [PATCH] arm64: configs: moto-holi: Enable CONFIG_IP_NF_TARGET_TTL

Enables the ability to execute a command
iptables -t mangle -A POSTROUTING -o <eth> -j TTL --ttl-set <TTL>

Change-Id: I30e139692e3f0e38c1026a052668bf7807871594
Signed-off-by: Sevenrock <sevenrock@hotmail.de>
---
 arch/arm64/configs/vendor/lineage_moto-holi.config | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/vendor/lineage_moto-holi.config b/arch/arm64/configs/vendor/lineage_moto-holi.config
index 97d672305b22..e73aefd1837d 100644
--- a/arch/arm64/configs/vendor/lineage_moto-holi.config
+++ b/arch/arm64/configs/vendor/lineage_moto-holi.config
@@ -45,5 +45,8 @@ CONFIG_THINLTO=y
 # Exfat
 CONFIG_EXFAT_FS=m
 
+# Net
+CONFIG_IP_NF_TARGET_TTL=y
+
 # Wifi
 CONFIG_QCA_CLD_WLAN=m
-- 
2.39.5

